collect.py

```diff
@@ -308,4 +307,4 @@ def main(bugs):
     random.shuffle(bugs)
     with ThreadPoolExecutor(max_workers=MAX_THREADS) as executor:
         for i in range(MAX_THREADS):
-            executor.submit(main, bugs[i::MAX_THREADS])
```

Comment:
I think the resulting code here is not valid Python, it's strange that it even runs.

Just remove the entire threadpoolexecutor block and call main(bugs).